.ahoy.yml

```
@@ -48,7 +48,7 @@ commands:
 
   drush:
     usage: Run drush commands in cli container.
-    cmd: docker-compose exec -T cli drush "$@"
+    cmd: docker-compose exec -e COLUMNS=120 -T cli drush "$@"
```

So I think lets go with that instead:

```suggestion
    cmd: docker-compose exec -e -T cli drush "$@"
```